typhon/config.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```

Comment:
Does it make a difference for a script without execution rights? 😉

Reply:
I'll remove the whole shebang. I don't think it ever makes sense to execute this.